src/diffusers/pipelines/unclip/pipeline_unclip.py

```diff
@@ -18,11 +18,11 @@
 import torch
 from torch.nn import functional as F
 
-from diffusers import PriorTransformer, UNet2DConditionModel, UNet2DModel
-from diffusers.pipeline_utils import DiffusionPipeline, ImagePipelineOutput
-from diffusers.schedulers import UnCLIPScheduler
 from transformers import CLIPTextModelWithProjection, CLIPTokenizer
 
+from ...models import PriorTransformer, UNet2DConditionModel, UNet2DModel
```

Comment:
@williamberman  inside `src/diffusers` we always use relative imports to make sure the pypi package is nicely self-contained.